dbms/src/Flash/Mpp/MPPTaskId.h

```diff
@@ -56,12 +64,15 @@ struct MPPQueryId
     String toString() const
     {
         return fmt::format(
-            "<query_ts:{}, local_query_id:{}, server_id:{}, start_ts:{}, resource_group: {}>",
+            "<query_ts:{}, local_query_id:{}, server_id:{}, start_ts:{}, resource_group: {}, conn_id: {}, conn_alias: "
+            "{}>",
```

Comment:
(not related to this PR)
Must we log the `resource_group` into logging? I think the MPPTaskId is quite long now. It may bring performance regression when there are lots of logging.

@guo-shaoge

Reply:
I agree, I think for information like `conn_id/conn_alias/resource_group`, we can only log it on receiving the request, not log it for every log of during the query